src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,37 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Parser {
+    public static String parseDate(String str) {
+        LocalDateTime date = LocalDateTime.parse(str, DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm"));
+        DateTimeFormatter dateFormat = DateTimeFormatter.ofPattern("MMM dd yyyy HHmm");
+        String displayDate = date.format(dateFormat);
+        return displayDate;
+    }
+
+    public static String retrieveDate(String str) {
+        LocalDateTime date = LocalDateTime.parse(str, DateTimeFormatter.ofPattern("MMM dd yyyy HHmm"));
+        DateTimeFormatter dateFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm");
+        String displayDate = date.format(dateFormat);
+        return displayDate;
+    }
+
+    public static String[] parseTask(String input) {
```

Comment:
I fear that this name might be misleading for some developers for the same reason stated in the above comment. Perhaps consider `extractArgsFromInput(String)`?